### PR TITLE
fix: Remove incorrect vertical gap for the cart empty state

### PR DESCRIPTION
**Pull request by filipewl**
## What's the purpose of this pull request?

A small patch to fix a minor visual bug with the cart's empty state.

## How does it work?

Before|After
-|-
Screen Shot 2022-05-02 at 15 08 03|Screen Shot 2022-05-02 at 15 07 31

## How to test it?

Open the cart and the vertical gap should be gone.

## References

- Mirrored from https://github.com/vtex-sites/gatsby.store/pull/20.

### CODE REVIEW

File: src/components/common/Navbar/navbar.scss [hunk 1]
```diff
@@ -47,6 +47,12 @@
     margin-left: calc(-1 * var(--fs-spacing-3));
   }
 
+  [data-fs-button-signin-link] + [data-fs-button-cart] {
```
Comment (filipewl) [inline src/components/common/Navbar/navbar.scss:50]: Before|After
-|-
Screen Shot 2022-05-03 at 15 38 05|Screen Shot 2022-05-03 at 15 37 51

Comment (hellofanny) [reply src/components/common/Navbar/navbar.scss:50]: Niceeeeeee!

**Comment by hellofanny** [comment]
@filipewl do we need to check anything else in this PR? Or can we merge it after solving the conflicts?

**Review by hellofanny — APPROVED** [review]

**Review by eduardoformiga — APPROVED** [review]
LGTM!

The CI didn't generate the preview correctly to test on build mode, though 🤔

**Comment by filipewl** [comment]
> LGTM!
> 
> The CI didn't generate the preview correctly to test on build mode, though 🤔

It should be working now: https://preview-11--nextjs.preview.vtex.app/!

CleanShot 2022-05-17 at 14 37 02

**Pull request merged**